src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/authorization/EligibleProviderMapper.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.authorization
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceProvider
+
+@Component
+class EligibleProviderMapper {
```

Comment:
i'm not super keen on the naming of some of these components. suggestions welcome.